### DISCUSSION TITLE
Capture parts of a node that are not included in another node

### DISCUSSION THREAD

**Discussion by pwntester** [nvim-treesitter/nvim-treesitter#2173] [Q&A]
Hi,

Im markdown, we have the following for fenced code blocks: 

```
fenced_code_block [43, 0] - [47, 0]
  info_string [43, 3] - [43, 7]
    language [43, 3] - [43, 7]
  code_fence_content [44, 0] - [46, 0]
```

I would like to capture the initial and ending three backticks which are not emitted by the parser (nor even as anonymous nodes). Is there a way to capture the characters that are part of `fenced_code_block` but not part of `code_fence_content` for example?

Similarly, for quote blocks, the parser emits:

```
block_quote [61, 0] - [62, 0]
  paragraph [61, 2] - [62, 0]
```

Where I would like to capture the initial `>`, that is the characters in block_quote that are not part of `paragraph`

Is this possible?

Thanks!

**Comment by pwntester** [comment]
I managed to get the highlight groups correct by capturing the outer node with the desired capture group (eg: `@punctuation.delimiters`) and the inner one with `@none`. Is there a native TS way of doing the same?

**Reply by theHamsta** [reply]
Not really. Unless you write your own directive what you want. the most straight forward thing would be to ask the parser creator to expose the  ` ``` ` token.
